src/mcts/node.cc

```diff
@@ -370,7 +370,7 @@ V3TrainingData Node::GetV3TrainingData(GameResult game_result,
   return result;
 }
 
-void NodeTree::MakeMove(Move move) {
+void NodeTree::MakeMove(Move move, bool no_tree_reuse) {
```

Comment:
Tree reuse has little to do with making a move, to this parameter doesn't seem like a good fit for this function.

Instead, I'd do a separate function NodeTree::TrimTreeAtHead() and would call it when I need to kill a tree.

Reply:
Done.